gcp-endpoint/load_balancer.tf

```diff
@@ -18,31 +18,33 @@ resource "google_compute_target_https_proxy" "main" {
   ssl_certificates = [google_compute_managed_ssl_certificate.main.id]
 }
 
-resource "google_compute_global_forwarding_rule" "main" {
+resource "google_compute_target_http_proxy" "main" {
+  name    = var.resource_name
+  url_map = google_compute_url_map.https_redirect.self_link
+}
+
+resource "google_compute_global_forwarding_rule" "http" {
   provider   = google-beta
   name       = var.resource_name
   labels     = var.labels
-  target     = google_compute_target_https_proxy.main.self_link
+  target     = google_compute_target_http_proxy.main.self_link
   ip_address = google_compute_global_address.main.address
-  port_range = "443"
+  port_range = "80"
```

Comment:
Is this change because we are doing ssl termination at the loadbalancer?

Reply:
I think this change is misleading. There was only one rule before (port 443), now there are two. One for port 80 and one for port 443. 80 is a redirect, 443 points to the same target proxy and backend as before. [Here's](https://github.com/massdriver-cloud/terraform-modules/blob/abeda7f43eedccdda9a14ccc9a8643163a72f111/gcp-endpoint/load_balancer.tf#L26-L44) the file w/o the diff.